test/suite/test_cc02.py

```diff
@@ -76,45 +79,38 @@ def test_cc(self):
             ',stable_timestamp=' + self.timestamp_str(100))
 
         # Check that the new updates are only seen after the update timestamp.
-        self.check(bigvalue2, uri, nrows, 100)
+        self.check(value2, uri, nrows, 100)
 
         # Load a slight modification with a later timestamp.
         self.large_modifies(uri, 'A', ds, 10, 1, nrows, 110)
         self.large_modifies(uri, 'B', ds, 20, 1, nrows, 120)
         self.large_modifies(uri, 'C', ds, 30, 1, nrows, 130)
 
         # Set of update operations with increased timestamp.
-        self.large_updates(uri, bigvalue, ds, nrows, 150)
+        self.large_updates(uri, value, ds, nrows, 150)
 
         # Set of update operations with increased timestamp.
-        self.large_updates(uri, bigvalue2, ds, nrows, 180)
+        self.large_updates(uri, value2, ds, nrows, 180)
 
         # Set of update operations with increased timestamp.
-        self.large_updates(uri, bigvalue, ds, nrows, 200)
-
-        # Check that the modifies are seen.
-        bigvalue_modA = bigvalue2[0:10] + 'A' + bigvalue2[11:]
-        bigvalue_modB = bigvalue_modA[0:20] + 'B' + bigvalue_modA[21:]
-        bigvalue_modC = bigvalue_modB[0:30] + 'C' + bigvalue_modB[31:]
-        self.check(bigvalue_modA, uri, nrows, 110)
-        self.check(bigvalue_modB, uri, nrows, 120)
-        self.check(bigvalue_modC, uri, nrows, 130)
+        self.large_updates(uri, value, ds, nrows, 200)
```

Comment:
I removed the checks below because this is not the purpose of the test and it is already checked in other tests.